bazel_codegen/lib/src/run_phases.dart

```diff
@@ -76,8 +87,16 @@ class _CodegenWorker extends AsyncWorkerLoop {
       var bazelRelativeInputs = request.inputs
           .map((input) => _bazelRelativePath(input.path, buildArgs.rootDirs));
 
-      logHandle = await _runBuilders(
-          builders, buildArgs, defaultContent, timings,
+      final srcPaths =
```

Comment:
lets move this code into a method so the error handling logic remains consistent etc

Reply:
Added a wrapping `_runBuilders` back in run_phases and shifted more of the Resource creation over there. The separate `runBuilders` is now a bit more focused and should be easier to test.

The bug I want to fix and test is still in `runBuilders`.